modules/misc/qt.nix

```diff
@@ -128,7 +156,7 @@ in {
     };
   };
 
-  config = mkIf (cfg.enable && cfg.platformTheme != null) {
+  config = mkIf cfg.enable {
```

Comment:
This should be a separate commit, I've already done this in #4312 fwiw.

Reply:
But this pr will become a nonsense if not removed here. And #4312 depends on this pr.

Reply:
Feel free to either split it into a commit of your own or cherry-pick mine. There is another change that should be made along with this one that I ought to have put in the same commit.

> And #4312 depends on this pr.

Not really. Only one of the changes depends on this to do anything, the rest doesn't (and there are already other qt6 themes in the module that currently don't work).